CkanConnector/package.json

```diff
@@ -4,9 +4,11 @@
   "description": "MAGDA CKAN Connector",
   "main": "index.js",
   "scripts": {
-    "build": "tsc",
+    "build": "export REGISTRY_URL=http://localhost:6100/;npm run generateSwaggerApis && tsc",
```

Comment:
nice, though this won't work on Windows because npm runs scripts under CMD even when it's invoked from bash.  That's why the `bash -c` stuff below.  But I'll fix it next time I run into it.